lib/stat_tracker.rb

```diff
@@ -1,5 +1,16 @@
+require 'csv'
 class StatTracker
-    def initialize
-        
+    attr_reader :game_teams,
+                :games,
+                :teams
+
+    def initialize(locations)
+      game_teams = CSV.read locations[:game_teams], headers: true, header_converters: :symbol
+      games = CSV.read locations[:games], headers: true, header_converters: :symbol
+      teams = CSV.read locations[:teams], headers: true, header_converters: :symbol
```

Comment:
Looks good, using .open really messed with us

Reply:
This was a struggle, .open was leaving us stumped.